services/minespace-web/tsconfig.json

```diff
@@ -26,6 +26,7 @@
     "esModuleInterop": true /* Emit additional JavaScript to ease support for importing CommonJS modules. This enables 'allowSyntheticDefaultImports' for type compatibility. */,
     "forceConsistentCasingInFileNames": true /* Ensure that casing is correct in imports. */,
     "strict": false /* Enable all strict type-checking options. */,
+    "strictNullChecks": false,
```

Comment:
whoops?

Reply:
Yeah whoops.  Not sure how that made it back in.